test/reparent.py

```diff
@@ -621,6 +621,60 @@ def test_reparent_with_down_slave(self, shard_id='0'):
     tablet.kill_tablets([tablet_62344, tablet_62044, tablet_41983,
                          tablet_31981])
 
+  def test_change_type_semi_sync(self):
+    utils.run_vtctl(['CreateKeyspace', 'test_keyspace'])
+
+    # create the database so vttablets start, as they are serving
+    tablet_62344.create_db('vt_test_keyspace')
+    tablet_62044.create_db('vt_test_keyspace')
+    tablet_41983.create_db('vt_test_keyspace')
+    tablet_31981.create_db('vt_test_keyspace')
+
+    # Start up a soon-to-be master, 3 replicas mysql and vttablet
```

Comment:
I couldn't quite understand this comment. Did you mean "Start up a soon-to-be master, 2 replicas and rdonly"?